frontend/src/modules/Pvt/settings/atoms/queryAtoms.ts

```diff
@@ -0,0 +1,61 @@
+import { PvtData_api } from "@api";
+import { apiService } from "@framework/ApiService";
+import { EnsembleIdent } from "@framework/EnsembleIdent";
+import { atomWithQueries } from "@framework/utils/atomUtils";
+import { UseQueryResult } from "@tanstack/react-query";
+
+import { selectedEnsembleIdentsAtom, selectedRealizationsAtom } from "./derivedAtoms";
+
+import { CombinedPvtDataResult } from "../../typesAndEnums";
+
+const STALE_TIME = 60 * 1000;
+const CACHE_TIME = 60 * 1000;
+
+export const pvtDataQueriesAtom = atomWithQueries((get) => {
+    const selectedEnsembleIdents = get(selectedEnsembleIdentsAtom);
+    const selectedRealizations = get(selectedRealizationsAtom);
+
+    const ensembleIdentsAndRealizations: { ensembleIdent: EnsembleIdent; realization: number }[] = [];
+    for (const ensembleIdent of selectedEnsembleIdents) {
+        for (const realization of selectedRealizations) {
+            ensembleIdentsAndRealizations.push({ ensembleIdent, realization });
+        }
+    }
+
+    const queries = ensembleIdentsAndRealizations
+        .map((el) => {
+            return () => ({
+                queryKey: ["tableData", el.ensembleIdent.toString(), el.realization],
```

Comment:
Not introduced in this PR, but this queryKey should probably be more specific. e.g. `pvtTableData` instead of `tableData`

Reply:
Good point, I forgot to take a second look at this. Changed to `pvtTableData`.